nerves_defconfig

```diff
@@ -27,9 +27,9 @@ BR2_ROOTFS_POST_BUILD_SCRIPT="${NERVES_DEFCONFIG_DIR}/post-build.sh ${BR2_EXTERN
 BR2_ROOTFS_POST_IMAGE_SCRIPT="${NERVES_DEFCONFIG_DIR}/post-createfs.sh"
 BR2_LINUX_KERNEL=y
 BR2_LINUX_KERNEL_CUSTOM_TARBALL=y
-BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="$(call github,raspberrypi,linux,raspberrypi-kernel_1.20200902-1)/linux-raspberrypi-kernel_1.20200902-1.tar.gz"
+BR2_LINUX_KERNEL_CUSTOM_TARBALL_LOCATION="$(call github,raspberrypi,linux,1.20210831)/linux-raspberrypi-kernel_1.20210831.tar.gz"
```

Comment:
Now I try this commit by cherry-picking to my custom_rpi3. The name of the tarball had to be `linux-1.20210831.tar.gz`.

`linux-raspberrypi-kernel_1.20210831.tar.gz` -> `linux-1.20210831.tar.gz`

Reply:
I found [new release](https://github.com/raspberrypi/linux/releases/tag/1.20210928).

If there is a test I am capable of, I am willing to do it.

Reply:
Thanks for letting me know about the latest release. It makes sense to use it. After ElixirConf, I'll update all of the Raspberry Pi systems with it. As for testing, anything that you do is helpful. Just knowing that the basics work like WiFi and Ethernet helps.

Reply:
OK, I'll wait for your update work. Then I will test WiFi, Ethernet, etc. and report back to you. Thank you.

Reply:
@pojiro Thank you for verifying that the new kernel works for you.